recipes/ftfy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ftfy" %}
+{% set version = "4.4.2" %}
```

Comment:
It appears as if the newest version is `5.1` on pypi. I think it's not compatible with the version of spacy that's attempting to be packaged. Once this is accepted, please be sure to update the recipe to keep up-to-date with the package's releases.